src/amo/components/SectionLinks/index.js

```diff
@@ -2,40 +2,71 @@ import classNames from 'classnames';
 import React from 'react';
 import { compose } from 'redux';
 import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
 
 import Link from 'amo/components/Link';
 import type { ViewContextType } from 'amo/reducers/viewContext';
+import { setClientApp } from 'core/actions';
 import {
   ADDON_TYPE_EXTENSION,
   ADDON_TYPE_THEME,
+  CLIENT_APP_ANDROID,
+  CLIENT_APP_FIREFOX,
   VIEW_CONTEXT_EXPLORE,
   VIEW_CONTEXT_HOME,
+  VIEW_CONTEXT_LANGUAGE_TOOLS,
 } from 'core/constants';
+import type { ApiStateType } from 'core/reducers/api';
 import translate from 'core/i18n/translate';
 import { visibleAddonType } from 'core/utils';
+import DropdownMenu from 'ui/components/DropdownMenu';
+import DropdownMenuItem from 'ui/components/DropdownMenuItem';
 
 import './styles.scss';
 
 
 type SectionLinksProps = {
   className?: string,
+  clientApp: string,
+  dispatch: Function,
   i18n: Object,
+  router: Object,
   viewContext: ViewContextType,
 }
 
 export class SectionLinksBase extends React.Component {
+  setClientApp = (event) => {
+    event.preventDefault();
+
+    const { dispatch, router } = this.props;
+
+    const clientApp = event.currentTarget.getAttribute('data-clientapp');
+
+    dispatch(setClientApp(clientApp));
+    router.push(event.currentTarget.getAttribute('href'));
+  }
+
   props: SectionLinksProps;
 
   render() {
-    const { className, i18n, viewContext } = this.props;
+    const { className, clientApp, i18n, viewContext } = this.props;
     const isExploring = [VIEW_CONTEXT_EXPLORE, VIEW_CONTEXT_HOME]
       .includes(viewContext);
 
+    let forBrowserNameText;
+    if (clientApp === CLIENT_APP_FIREFOX) {
+      forBrowserNameText = i18n.gettext('for Firefox');
+    } else if (clientApp === CLIENT_APP_ANDROID) {
+      forBrowserNameText = i18n.gettext('for Firefox for Android');
+    }
+
     return (
       <ul className={classNames('SectionLinks', className)}>
         <li>
           <Link
-            className={classNames('SectionLinks-link', { 'SectionLinks-link--active': isExploring })}
+            className={classNames('SectionLinks-link', 'SectionLinks-explore',
+              { 'SectionLinks-link--active': isExploring }
+            )}
```

Comment:
Can we format this block like this?

```
className={classNames('SectionLinks-link', 'SectionLinks-explore', {
  'SectionLinks-link--active': isExploring,
})
```

I find it more readable and the diff will be slightly better if we need to add a new class in the future.